bottlerocket-settings-models/settings-extensions/kubelet-device-plugins/src/lib.rs

```diff
@@ -75,6 +77,13 @@ mod test {
                         rename_by_default: Some(true),
                         fail_requests_greater_than_one: Some(true),
                     }),
+                    device_partitioning_strategy: Some(NvidiaDevicePartitioningStrategy::MIG),
+                    mig: Some(NvidiaMigSettings {
+                        profile: Some(HashMap::from([(
+                            NvidiaGpuModel::try_from("a100.40gb").unwrap(),
+                            MigProfile::try_from("1g.5gb").unwrap()
+                        )]))
+                    }),
                 }),
```

Comment:
We should probably bump the version of `settings-extension-kubelet-device-plugins` to `0.2.0`.